src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -214,6 +246,16 @@ public virtual Publisher.IPublisherClient GrpcClient
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Adds one or more messages to the topic. Returns `NOT_FOUND` if the topic
```

Comment:
It is not clear to me what `NOT_FOUND` means in this context, is it an enum or is it literally the string "NOT_FOUND"? Or is this too low in the stack and I should not worry about this?